seed/models/analyses.py

```diff
@@ -0,0 +1,21 @@
+
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from seed.landing.models import SEEDUser as User
+
+
+class Analysis(models.Model):
+
+    BSYNCR = 1
+
+    ANALYSIS_TYPES = (
+        (BSYNCR, 'BSyncr'),
+    )
+
+    name = models.CharField(max_length=255, blank=False, default=None)
```

Comment:
Does `blank=False` and `default=None` kind of contradict here? I think we can at least remove blank here as it defaults to False
https://docs.djangoproject.com/en/3.1/ref/models/fields/#blank